<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>talkform</class>
 <widget class="QWidget" name="talkform">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1119</width>
    <height>520</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>新宋体</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="Line" name="form_line1">
   <property name="geometry">
    <rect>
     <x>304</x>
     <y>30</y>
     <width>16</width>
     <height>471</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QWidget" name="talkshow_widget" native="true">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>761</width>
     <height>509</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>400</y>
      <width>311</width>
      <height>110</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>新宋体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <layout class="QVBoxLayout" name="ChatGPT_layout">
     <item>
      <widget class="QTextEdit" name="ChatGPTinput_textedit">
       <property name="font">
        <font>
         <family>新宋体</family>
         <pointsize>13</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255, 170);</string>
       </property>
       <property name="placeholderText">
        <string>输入内容</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="ChatGPT_button_layout">
       <item>
        <spacer name="ChatGPT_button_layout_sp1">
         <property name="font">
          <font>
           <family>新宋体</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="ChatGPTnew_button">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="font">
          <font>
           <family>新宋体</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>新对话</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ChatGPTvoice_button">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="font">
          <font>
           <family>新宋体</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string> 语音 </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ChatGPTinput_button">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="font">
          <font>
           <family>新宋体</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string> 发送 </string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="imagelabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>761</width>
      <height>509</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>新宋体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextBrowser" name="ChatGPToutput_browser">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>9</y>
      <width>309</width>
      <height>391</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>新宋体</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 170);</string>
    </property>
   </widget>
   <zorder>imagelabel</zorder>
   <zorder>ChatGPToutput_browser</zorder>
   <zorder>verticalLayoutWidget_3</zorder>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>281</width>
     <height>511</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <layout class="QVBoxLayout" name="control_layout">
    <item>
     <widget class="QLabel" name="tips_label">
      <property name="font">
       <font>
        <family>新宋体</family>
        <pointsize>16</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>AI-虚拟人
声明：对话内容由AI生成，完全虚构，与现实无关。真实性自行分辨。
在下方输入您的OpenAI-API-Key，虚拟人配置文件，然后点击“启动”↓</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="control_layout_sp1">
      <property name="font">
       <font>
        <family>新宋体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="API_key_label">
      <property name="font">
       <font>
        <family>新宋体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>OpenAI-API-Key：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="API_Key_lineedit">
      <property name="font">
       <font>
        <family>新宋体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::PasswordEchoOnEdit</enum>
      </property>
      <property name="placeholderText">
       <string>sk-XXXXXXXXXXXXXXX</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="settingfile_label">
      <property name="font">
       <font>
        <family>新宋体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>虚拟人配置：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="setting_lineedit">
      <property name="font">
       <font>
        <family>新宋体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>.\userdata.json</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="talkoutput_browser">
      <property name="font">
       <font>
        <family>新宋体</family>
        <pointsize>13</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="contorl_button_layout" stretch="0,0,0">
      <item>
       <spacer name="contorl_button_layout_sp1">
        <property name="font">
         <font>
          <family>新宋体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="talkstart_button">
        <property name="font">
         <font>
          <family>新宋体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string> 启动 </string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="contorl_button_layout_sp2">
        <property name="font">
         <font>
          <family>新宋体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="control_layout_sp3">
      <property name="font">
       <font>
        <family>新宋体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="back_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1119</width>
     <height>520</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/image/xiaomeng.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="back_label2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1119</width>
     <height>520</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 130);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="select_json_button">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>271</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <zorder>back_label</zorder>
  <zorder>back_label2</zorder>
  <zorder>form_line1</zorder>
  <zorder>talkshow_widget</zorder>
  <zorder>verticalLayoutWidget</zorder>
  <zorder>select_json_button</zorder>
 </widget>
 <resources>
  <include location="NightRain.qrc"/>
 </resources>
 <connections/>
</ui>
